Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/DataStream/DataSource/Data/AbstractData.cs

```diff
@@ -55,5 +55,17 @@ public void ReleaseAsync(JobHandle releaseAccessDependency)
         {
             m_AccessController.ReleaseAsync(releaseAccessDependency);
         }
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Not really on topic for this PR but does this actually work since the method captures a private class member?

Reply:
Not sure, made an issue to investigate: #154